scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -73,6 +75,8 @@ class Router(formatOps: FormatOps) {
     val leftOwner = owners(formatToken.left)
     val rightOwner = owners(formatToken.right)
     val newlines = newlinesBetween(formatToken.between)
+    val isDef = rightOwner.tokens.headOption.exists(_.is[KwDef])
```

Comment:
Let's remove this line. Suggested solution below.

BTW. This returns true for any class/trait/object that contains methods.